src/components/Results/DetectedIssueResources.tsx

```diff
@@ -25,6 +34,11 @@ const DetectedIssueResources: React.FC<Props> = ({ detectedIssue }) => {
     <div>
       {guidanceResponseArray.map((response: R4.IGuidanceResponse, index: number) => {
         const guidanceResponseId = fhirpath.evaluate(response, 'id');
+        const codeFilters = fhirpath.evaluate(response, 'dataRequirement.codeFilter');
+        const link = 'http://hl7.org/fhir/';
+        const valueSetObj = codeFilters.find((cf: any) => cf.valueSet);
+        const codeFilterArray = codeFilters.filter((cf: any) => !!!cf.valueSet);
```

Comment:
Is the triple `!!!` handling some unique case for this code? Is there a reason this is used over a single one? (pardon the ignorance, Javascript it weird

Reply:
I believe I saw it online. I just changed it to a single one and tested it and it works fine so I'll make it a single one!